packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -217,24 +225,44 @@ const CFDPersonalDetailsForm = ({
     form_error,
 }: TCFDPersonalDetailsFormProps) => {
     const account_opening_reason = getAccountOpeningReasonList();
+    const { jurisdiction_selected_shortcode } = useCfdStore();
+    const {
+        client: { account_settings },
+    } = useStore();
+
+    const residence = residence_list?.find(item => item.text === account_settings?.residence);
+    const { data, isLoading: is_landing_company_details_loading } = useLandingCompanyDetails({
+        // @ts-expect-error jurisdiction_selected_shortcode type should be updated in cfd-store types
+        landing_company_details: jurisdiction_selected_shortcode,
+        country: residence?.value,
+    });
+
+    const tin_not_mandatory = data?.tin_not_mandatory === 1;
 
```

Comment:
i see we are checking !tin_not_mandatory everywhere in this file, maybe we can declare 
  const tin_mandatory = data?.tin_not_mandatory === 0; and reuse it .

Reply:
I just wanted to keep the flag in sync as we're getting from BE. The name should've been like `is_tin_mandatory`

Reply:
If we rename it, it'll need to negate every condition implemented in the code. If you prefer we can rename it however the card is in Progress QA.